packages/common/src/eips.ts

```diff
@@ -478,4 +478,12 @@ export const eipsDict: EIPsDict = {
   7864: {
     minimumHardfork: Hardfork.London,
   },
+  /**
+   * Description : Meter Contract Code Size And Increase Limit
+   * URL         : https://eips.ethereum.org/EIPS/eip-7907
+   * Status      : Draft
+   */
+  7907: {
+    minimumHardfork: Hardfork.Osaka,
```

Comment:
This EIP requires 3860. You can then also remove the `|| common.isActivedEIP(7907)` because if 7907 is active, then 3860 needs also to be active